Formula/httpd.rb

```diff
@@ -18,14 +18,188 @@ class Httpd < Formula
   depends_on "pcre"
 
   def install
-    # use Slackware-FHS layout as it's closest to what we want.
-    # these values cannot be passed directly to configure, unfortunately.
-    inreplace "config.layout" do |s|
-      s.gsub! "${datadir}/cgi-bin", "#{pkgshare}/cgi-bin"
-      s.gsub! "${datadir}/error",   "#{pkgshare}/error"
-      s.gsub! "${datadir}/htdocs",  "#{pkgshare}/htdocs"
-      s.gsub! "${datadir}/icons",   "#{pkgshare}/icons"
-    end
+    # point config files to opt_prefix instead of the version-specific prefix
+    inreplace "Makefile.in",
+      '#@@ServerRoot@@#$(prefix)#', '#@@ServerRoot@@'"##{opt_prefix}#"
+
+    # fix default user/group when running as root
+    inreplace "docs/conf/httpd.conf.in", /(User|Group) daemon/, "\\1 _www"
+
+    Pathname.new("docs/conf/httpd.conf.in").append_lines <<-EOS.undent
+      # Welcome page
+      Include #{etc}/httpd/extra/httpd-welcome.conf
+
+      # Additional configuration
+      IncludeOptional #{etc}/httpd/other/*.conf
+    EOS
+
+    Pathname.new("docs/conf/extra/httpd-welcome.conf").write <<-EOS.undent
+      # This enables the default Welcome page if there
```

Comment:
Is there no file like this included in the tarball?

Reply:
No, it's not included. I mostly lifted it from the Fedora rpm.

Reply:
probably best to leave it out in that case

Reply:
Let's pass on this for now